homeassistant/components/template/cover.py

```diff
@@ -318,7 +319,7 @@ def current_cover_tilt_position(self) -> int | None:
         """
         return self._tilt_value
 
-    @property
```

Comment:
It looks like this can be set in the constructor in a separate PR

Reply:
#100301